### PR TITLE
fix(test): v2 searchable aggregate test

### PR DESCRIPTION
**Pull request by sundersc**
Fix for Searchable v2 flaky test.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by lazpavel — APPROVED** [review]
LGTM

File: packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformerV2.e2e.test.ts [hunk 1]
```diff
@@ -149,6 +151,15 @@ test('query for aggregate scalar results', async () => {
     {},
   );
   expect(searchResponse).toBeDefined();
+
+  let searchPosts = searchResponse.data.searchPosts.items;
+  let expectedValue = searchPosts[0].ups;
```
Comment (yuth) [inline packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformerV2.e2e.test.ts:156]: We should not be relying on the first result and compare that value. It should be a know value (before hand) like you had it earlier. Can we just wait for all the data to be propagated before running this test

**Review by yuth — CHANGES_REQUESTED** [review]

File: packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformerV2.e2e.test.ts (hunk 1 above)
Comment (sundersc) [reply packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformerV2.e2e.test.ts:156]: Not relying on the first value, running the searchPosts query and calculating MIN value from the results and verifying it against the aggregate result.

I agree that waiting for the data to be propagated is the best solution, will work on it in a separate PR.

**Comment by sundersc** [comment]
Followed a different approach to make sure the data propagation is complete.
https://github.com/aws-amplify/amplify-cli/pull/8251

**Pull request closed without merging**